target/arc/semfunc.c

```diff
@@ -711,4 +711,57 @@ arc_gen_VPACK2HM(DisasCtxt *ctx, TCGv a, TCGv b, TCGv c)
   tcg_temp_free(c_h1);
 
   return DISAS_NEXT;
+}
+
+MemOp
+arc_gen_atld_op(DisasCtxt *ctx, TCGv_i32 b, TCGv c)
```

Comment:
Please add also here a comment on what to function supposedly should do.

Reply:
There already was a comment in header file, but it was lacking information. I added more comments as well as improving that one.